me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -16,12 +29,15 @@
   private static final String SIGN_TYPE_KEY = "signType";
   private static final String USER_TOKEN_KEY = "user_token";
 
+  private static String idMeWebVerifyGetAccessTokenUri;
   private static String idMeWebVerifyGetAuthUri;
   private static String idMeWebVerifyGetUserProfile;
 
   private static AccessTokenManager accessTokenManager;
-  private static String clientID;
-  private static String redirectURI = "";
+  private static RefreshAccessTokenHandler refreshAccessTokenHandler;
+  private static String clientId;
+  private static String redirectUri = "";
```

Comment:
why some fields are initialized with default values (empty) and others don't? Could you use the same approach for all of them?